test/release/examples/primers/modules.chpl

```diff
@@ -0,0 +1,390 @@
+     modToUse is a submodule of the overarching file module "modules".
+  */
+
+
+
+  /* Often, the functionality of a group of code doesn't need to be completely
+     visible to outside clients.  Sometimes it would be harmful for the client
+     to access a function that isn't part of the blessed API, and sometimes it
+     just adds unnecessary clutter to the namespace.  To avoid these cases, a
+     symbol can be declared "private" - this means that only code defined within
+     the same scope as the definition of this symbol can access it.
+  */
+
+  /* Here, hiddenFoo is a private global variable, which is only accessible by
+     symbols defined within modToUse
+  */
+  private var hiddenFoo = false;
+
+  /* hiddenBaz is a private function, which is also only accessible by symbols
+     defined within modToUse.
+  */
+  private proc hiddenBaz(a) {
+    writeln(a);
+    return a + 3;
+  }
+
+  module Inner2 {
+    /* Since the module Inner2 is defined within modToUse, it can access the
+       private variable hiddenFoo and the private function hiddenBaz.  However,
+       any private symbol defined within Inner2 will not be visible to symbols
+       defined outside of Inner2.
+     */
+
+    private var innerOnly = -17;
+    var canSeeHidden = !hiddenFoo;
+  }
+
+  // At the moment, private cannot be applied to type definitions; type
+  // aliases, and declarations of enums, records, and classes cannot be declared
+  // private.  Private also cannot be applied to fields or methods yet.
+
+}
+
+module AnotherModule {
+  var a = false;
+}
+
+module ThirdModule {
+  var b = -13.0;
+}
+
+
+module Conflict {
+  /* This parenthesis-less function shares a name with a symbol in modToUse. */
+  proc bar {
+    writeln("In Conflict.bar");
+    return 5;
+  }
+
+  var other = 5.0 + 3i;
+}
+
+
+proc main() {
+  writeln("Access from outside a module");
+
+  /* Often, it is desirable for the contents of a module to be accessed from
+     outside that context.  There are several strategies for accomplishing this:
+
+     First, if only one symbol defined in the module is desired, and it is
+     desired only once, the symbol can be referenced explicitly - this is done
+     using the module name and a separating '.' as a prefix to the name of the
+     symbol desired.
+  */
+  var thriceFoo = 3 * modToUse.foo; // should be 36
+  writeln(thriceFoo);
+
+
+  {
+    /* If many of the module's symbols are desired, or the same symbol is
+       desired multiple times, then it is best to utilize what is known as a
+       "use statement".
+    */
+    use modToUse;
+
+    /*
+      Use statements can be inserted at the global scope, as well as any
+      arbitrary inner scope defined.
+    */
+
+    /* A use statement brings all of the visible symbols of a module into the
+       scope which defines the use statement.  These symbols may then be
+       accessed without the module name prefix.
```

Comment:
"defines" -> "contains"

We've talked before about avoiding phrases implying that the symbols are injected into the current scope for fear of confusion w.r.t. shadowing and the like.  To that end, could this be rephrased to say something like "...makes all of a module's public symbols visible from within the scope that contains the use statement." to avoid that implication?